### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zlydh**
Update CHANGELOG for M30 release.

### CODE REVIEW

File: Firebase/Messaging/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Unreleased
+# 2018-07-31 -- v3.1.0
+- Added function to support grobal data flag
```
Comment (morganchen12) [inline Firebase/Messaging/CHANGELOG.md]: Consider rephrasing this to "Added support for global Firebase data collection flag." Also, include the PR number at the end of the line, if possible.

Comment (paulb777) [reply Firebase/Messaging/CHANGELOG.md]: s/grobal/global/

Comment (zlydh) [reply Firebase/Messaging/CHANGELOG.md]: Done

**Comment by paulb777** [comment]
I'll merge this, so it's part of the release branch.

**Pull request merged**